css/styles.css

```diff
@@ -10176,6 +10176,13 @@ header.masthead  {
   padding-top: 20px;
 }
 
+@media (min-width: 768px) {
```

Comment:
I think we can fix this more elegantly by, in call_to_action.html, adding a second row with the same columns as the first one, and move the buttons into that. I would prefer to at least take a quick stab at getting that working before resorting to this hack. Feel free to either take a stab at that, or back this change out of this PR and open a separate one in a branch I can play in.

Reply:
Actually that is not the right fix, because it will break on mobile, but I think I can figure out the right fix ;) I guess it's fine to go ahead and merge this way if you want though, and I can post a PR to replace it if I want.

Reply:
The challenge there is when the content collapses in a narrow / mobile layout -- you want the buttons tucked right under that text in mobile. I'm sure there's still a less gross fix here, but I'll merge this and we can play with it after, easy enough to drop this bit.